e2e/env/.env.example

```diff
@@ -18,14 +18,9 @@
 # resource, will contain that *lowercased* username in their URLs.
 #
 
-E2E_TEST_ESS_POD=https://pod.inrupt.com/<username-in-lowercase>/
-E2E_TEST_ESS_IDP_URL=https://broker.pod.inrupt.com
-E2E_TEST_ESS_REFRESH_TOKEN=<refresh token>
-E2E_TEST_ESS_CLIENT_ID=<client id>
-E2E_TEST_ESS_CLIENT_SECRET=<client secret>
-
-E2E_TEST_ESS_COMPAT_POD=https://pod-compat.inrupt.com/<username-in-lowercase>/
-E2E_TEST_ESS_COMPAT_IDP_URL=https://broker.pod-compat.inrupt.com
-E2E_TEST_ESS_COMPAT_REFRESH_TOKEN=<refresh token>
-E2E_TEST_ESS_COMPAT_CLIENT_ID=<client id>
-E2E_TEST_ESS_COMPAT_CLIENT_SECRET=<client secret>
+E2E_TEST_IDP=https://openid.dev-next.inrupt.com/
+E2E_TEST_ENVIRONMENT="ESS Dev-Next"
+E2E_TEST_VC_PROVIDER=https://vc.dev-next.inrupt.com/
+E2E_TEST_VC_SUBJECT=https://id.dev-next.inrupt.com/inrupte2etestspodspacesdevnext
```

Comment:
```suggestion
```

`E2E_TEST_VC_SUBJECT` is no longer used, as we use the `session.info.webid` instead